app/views/ops/_ap_form_file.html.haml

```diff
@@ -13,8 +13,8 @@
           %td
             .rollover
               = image_tag("/images/toolbars/new.png", :class => "rollover small")
-          %td=_("<New Entry>")
-          %td=_("<New Entry>")
+          %td= _("<New Entry>")
+          %td= _("<New Entry>")
```

Comment:
This probably shouldn't be changed in this PR, but would be better, if `<` and `>` brackets were outside of gettext calling, as it is used for exapmle [here](https://github.com/ManageIQ/manageiq/blob/master/app/views/miq_ae_customization/_dialog_field_form.html.haml#L33)